tools/accuracy_checker/configs/efficientnet-b7-pytorch.yml

```diff
@@ -0,0 +1,95 @@
+models:
+  - name: efficientnet-b7-pytorch
+
+    launchers:
+      - framework: onnx_runtime
+        model: efficientnet-b7-pytorch/efficientnet-b7-pytorch.onnx
+        adapytorcher: classification
+        cpu_extensions: AUTO
+        inputs:
+          - name: data
+            type: INPUT
+            shape: 1,3,600,600
+
+
+    datasets:
+      - name: imagenet_1000_classes
+        data_source: ImageNet
+        reader: pillow_imread
+
+        preprocessing:
+          - type: resize
+            size: 632
+            aspect_ratio_scale: greater
+            use_pillow: True
+            interpolation: BICUBIC
+            # Crop ratio 0.949
+          - type: crop
+            use_pillow: True
+            size: 600
+
+          - type: normalization
+            mean: (123.675,116.28,103.53)
+            std: (58.395,57.12,57.375)
+
+        metrics:
+          - name: accuracy@top1
+            type: accuracy
+            top_k: 1
+            reference: 84.42
+            threshold: 0.05
+          - name: acciracy@top5
+            type: accuracy
+            top_k: 5
+            reference: 96.91
```

Comment:
ref numbers should be commented or removed. It can get false positive in light accuracy validation.
BTW, you already mention them in model docs, it is not necessary to additionaly store them in AC config.